### PR TITLE
Anglara - Update job-openings-data.json

### PR DESCRIPTION
**Pull request by dhavalkakkad**

### CODE REVIEW

File: public/data/job-openings-data.json [hunk 1]
```diff
@@ -445,6 +445,43 @@
...
+          "count": 1,
+          "experience": "2 to 3 Years experience"
+        },
+        {
+          "title": "QA Interns",
+          "count": 2,
+          "experience": "NA"
+        }
+      ]
+    },
```
Comment (piedcipher) [inline public/data/job-openings-data.json:485]:
```suggestion
    }
```

Comment (piedcipher) [reply public/data/job-openings-data.json:485]: Removing trailing comma should fix ci error

Comment (JaldeepAsodariya) [reply public/data/job-openings-data.json:485]: @dhavalkakkad Please check @piedcipher review feedback

**Comment by JaldeepAsodariya** [comment]
@dhavalkakkad It should have only your change (Your new object addition)

**Comment by JaldeepAsodariya** [comment]
Closing this PR, Duplicate of #432

**Pull request closed without merging**